tests/phpunit/elementor/modules/home/transformations/test-remove-sidebar-upgrade-for-pro-users.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Modules\Home\Transformations;
+
+use Elementor\Modules\Home\Transformations\Remove_Sidebar_Upgrade_For_Pro_Users;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Remove_Sidebar_Upgrade_For_Pro_Users extends Elementor_Test_Base {
+
+	public function test_core_plugin() {
```

Comment:
name should be `test_{test_function_name}__{scenario}`

Reply:
so it should be something like `test_transform__core(has_core)`

Reply:
Updated.